### PR TITLE
Contributing.md beatrix droid

### PR DESCRIPTION
**Pull request by Beatrix-droid**

### CODE REVIEW

**Comment by Beatrix-droid** [comment]
updated the contribution.md file as requested, and created another markdown where the guide on git is, and linked it to the contribution,md file. This makes the contribution.md file less cluttered, as that guide is now gone, but at the same time, users that will need the guidance to make a PR will be pointed to the right resources.

File: git-guidance.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Click here for a <a href="https://www.youtube.com/watch?v=c6b6B9oN4Vg">video</a> explaining steps below please.
+
+ - Fork the below github repository  <br/>
+      <code> https://github.com/qiskit-community/ecosystem </code>
+ - After the Fork and Clone the github repository in your local system  <br />
+      <code> git clone git@github.com:<your_user_name>/ecosystem.git </code>
+ - Keep your fork up-to-date  <br/>
+      <code> git remote add upstream https://github.com/qiskit-community/ecosystem.git </code>
+ - Checkout the Master branch and merge the up-stream   <br/>
+      <code> git checkout master </code>   <br/>
```
Comment (IceKhan13) [inline git-guidance.md:14]: Can we rename branches from `master` to `main`?

**Review by IceKhan13 — CHANGES_REQUESTED** [review]
Love it! 

Let's move git guide to docs folder, rename branch and we are ready to merge.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+First read the overall project contributing guidelines. These are all
+included in the qiskit documentation:
+
+https://qiskit.org/documentation/contributing_to_qiskit.html
+
+## Joining the Ecosystem
+To join ecosystem you need to create 
+[submission issue](https://github.com/qiskit-community/ecosystem/issues/new?labels=&template=submission.yml&title=%5BSubmission%5D%3A+) 
+and fill in all required details. That's it!
```
[the remark this answers is not in the document]
Comment (Beatrix-droid) [reply CONTRIBUTING.md:11]: Noted, will make the changes !

**Comment by Beatrix-droid** [comment]
i have closed this pull request and opened another one with the changes that were requested

**Comment by IceKhan13** [comment]
Linking new PR here https://github.com/qiskit-community/ecosystem/pull/182

**Pull request closed without merging**